mysql/tests/compose/mysql.conf

```diff
@@ -0,0 +1,5 @@
+[mysqld]
+# log_short_format=ON  # Uncomment to simulate short log format.
+slow_query_log = 1
+slow_query_log_file = /var/log/mysql/slow.log
```

Comment:
Could you add a comment why we are configuring the slow logs? Basically, what you put in the Motivation section of the PR.